interpreter/cling/tools/packaging/cpt.py

```diff
@@ -1923,7 +1923,8 @@ def custom_input(prompt, always_yes=False):
             Do you want to continue? [yes/no]: ''', args['y']).lower()
         if choice in yes:
             subprocess.call(
-                "sudo {0} -m pip install distro".format(sys.executable), shell=True
+                "sudo {0} -m pip" +
+                " install distro".format(sys.executable), shell=True
```

Comment:
I'd suggest to split at the `,` to avoid breaking the string literal (same below, where it applies).